test/e2e-tests.sh

```diff
@@ -35,19 +48,41 @@ if [ "$(uname)" == "Darwin" ]; then
   grep=ggrep
 fi
 
+# Use GNU Tools on MacOS (Requires the 'grep' and 'gnu-sed' Homebrew formulae)
+if [ "$(uname)" == "Darwin" ]; then
+  sed=gsed
+  grep=ggrep
+fi
+
 # Eventing main config path from HEAD.
 readonly EVENTING_CONFIG="./config/"
 readonly EVENTING_MT_CHANNEL_BROKER_CONFIG="./config/brokers/mt-channel-broker"
 readonly EVENTING_IN_MEMORY_CHANNEL_CONFIG="./config/channels/in-memory-channel"
 
-# Vendored eventing test iamges.
+# Vendored Eventing Test Images.
 readonly VENDOR_EVENTING_TEST_IMAGES="vendor/knative.dev/eventing/test/test_images/"
 # HEAD eventing test images.
 readonly HEAD_EVENTING_TEST_IMAGES="${GOPATH}/src/knative.dev/eventing/test/test_images/"
 
 # Config tracing config.
 readonly CONFIG_TRACING_CONFIG="test/config/config-tracing.yaml"
 
+# Strimzi Kafka Cluster Brokers URL (base64 encoded value for k8s secret)
+readonly STRIMZI_KAFKA_NAMESPACE="kafka" # Installation Namespace
+readonly STRIMZI_KAFKA_CLUSTER_BROKERS="my-cluster-kafka-bootstrap.kafka.svc:9092"
+readonly STRIMZI_KAFKA_CLUSTER_BROKERS_ENCODED="bXktY2x1c3Rlci1rYWZrYS1ib290c3RyYXAua2Fma2Euc3ZjOjkwOTI=" # Simple base64 encoding of STRIMZI_KAFKA_CLUSTER_BROKERS
+
+# Eventing Kafka main config path from HEAD.
+readonly KAFKA_CRD_CONFIG_TEMPLATE_DIR="./config/channel"
+readonly DISTRIBUTED_TEMPLATE_DIR="${KAFKA_CRD_CONFIG_TEMPLATE_DIR}/distributed"
+readonly CONSOLIDATED_TEMPLATE_DIR="${KAFKA_CRD_CONFIG_TEMPLATE_DIR}/consolidated"
+
+# Eventing Kafka Channel CRD Secret (Will be modified with Strimzi Auth)
+readonly EVENTING_KAFKA_SECRET_TEMPLATE="300-kafka-secret.yaml"
+
```

Comment:
is comment correct?  I thought it wasn't using auth for this?

Reply:
Indeed; modified comment to reflect this